### DISCUSSION TITLE
Custom component dynamically loading content triggers Editor's "update" event

### DISCUSSION THREAD

**Discussion by sarkarology** [ueberdosis/tiptap#2900] [General Questions]
I've built a custom image extension to load images from Supabase. The images are loaded dynamically since Supabase doesn't allow direct linking.

Everything is working, the only problem is, when the component downloads the image (on parsing), the src attribute of the img tag changes, and it triggers the Editor's update event.

Is there a way to prevent triggering update event on dynamically loading images?

**Comment by shahsagarmm** [comment]
Hi, did you find any solution?
